### PR TITLE
fix(linkis-dist): optimized the install-kubernetes.sh script

### PR DESCRIPTION
**Pull request by jacktao007**
### What is the purpose of the change

optimized the install-kubernetes.sh script.

### Brief change log

- optimized the install-kubernetes.sh script;

### Checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [ ] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [Linkis mailing list](https://linkis.apache.org/community/how-to-subscribe) first)
- [ ] **If this is a code change**: I have written unit tests to fully verify the new behavior.

### CODE REVIEW

File: linkis-dist/bin/install-kubernetes.sh [hunk 1]
```diff
@@ -14,16 +14,182 @@
...
+}
+#change the label
+tag(){
+    docker tag  $MIRRORS/apache/incubator-linkis/linkis:latest linkis:dev
+    docker tag  $MIRRORS/apache/incubator-linkis/linkis-web:latest linkis-web:dev
+    docker tag  $MIRRORS/apache/incubator-linkis/linkis-ldh:latest linkis-ldh:dev
+}
+#create an image to carry mysql
+make-linikis-image-with-mysql-jdbc(){
+    ../docker/scripts/make-linikis-image-with-mysql-jdbc.sh
```
Comment (AaronLinOops) [inline linkis-dist/bin/install-kubernetes.sh:58]: 使用 ${WORK_DIR}/linkis-dist/docker/... 这样的路径是不是更好一些？

File: linkis-dist/bin/install-kubernetes.sh [hunk 2]
```diff
@@ -14,16 +14,182 @@
 # limitations under the License.
 #
```
Comment (AaronLinOops) [inline linkis-dist/bin/install-kubernetes.sh:16]: 脚本名称为 install-kubernetes.sh 给人感觉像是要安装 k8s ，是否有更合适一些的名称？

File: linkis-dist/bin/install-kubernetes.sh [hunk 3]
```diff
@@ -14,16 +14,182 @@
...
+    echo "example:"
+    echo "./install-kubernetes.sh pull                                   pull image with ghcr.io"
+    echo "./install-kubernetes.sh pull -m ghcr.dockerproxy.com           pull image with ghcr.dockerproxy.com"
+    echo "./install-kubernetes.sh install                                install linkis to kind and kubernetes"
+    echo "./install-kubernetes.sh pull-install -m ghcr.dockerproxy.com   pull image and install linkis to kind and kubernetes"
+}
+
+#pull the container image of the linkis
+pull(){
+    docker pull $MIRRORS/apache/incubator-linkis/linkis-ldh:latest
```
Comment (AaronLinOops) [inline linkis-dist/bin/install-kubernetes.sh:46]: tag名称是否可以参数化，指定具体的tag，默认是 latest ？

File: linkis-dist/bin/install-kubernetes.sh [hunk 4]
```diff
@@ -14,16 +14,182 @@
...
+        echo $(date "+%Y-%m-%d %H:%M:%S") "debug: "$1
+    fi
+}
+
+info(){
+    echo $(date "+%Y-%m-%d %H:%M:%S") "info: "$1
+}
+
+
+check-docker(){
```
Comment (casionone) [inline linkis-dist/bin/install-kubernetes.sh:107]: function name with `-` may casue error about not a valid identifier 
can replace with`_`

File: linkis-dist/bin/install-kubernetes.sh (hunk 1 above)
Comment (jacktao007) [reply linkis-dist/bin/install-kubernetes.sh:58]: 好的

File: linkis-dist/bin/install-kubernetes.sh (hunk 3 above)
Comment (jacktao007) [reply linkis-dist/bin/install-kubernetes.sh:46]: 好的

File: linkis-dist/bin/install-kubernetes.sh (hunk 4 above)
Comment (jacktao007) [reply linkis-dist/bin/install-kubernetes.sh:107]: 好的

**Review by casionone — APPROVED** [review]

**Pull request merged**